crates/bevy_app/src/app.rs

```diff
@@ -257,6 +263,35 @@ impl App {
             sub_app.run();
         }
 
+        // move schedules out of the world so we can modify it as we iterate
+        // through the schedule events
+        let mut schedules = self
+            .world
+            .remove_resource::<Schedules>()
+            .expect("No Schedules resource in world");
```

Comment:
```suggestion
            .unwrap();
```

This isn't any better than the stock error message